app/controllers/test/mock_socure_controller.rb

```diff
@@ -45,6 +45,21 @@ def docv_results
       render json: DocAuth::Mock::Socure.instance.selected_fixture_body
     end
 
+    def image_request_endpoint
```

Comment:
the mock socure endpoints are [not currently in use](https://gsa-tts.slack.com/archives/C05HSH9RQ57/p1748374799702529?thread_ts=1748374380.920279&cid=C05HSH9RQ57), but i wanted to keep my work in lockstep with it, in case it was revived.